layout/src/markup/projects/index.hbs

```diff
@@ -22,23 +22,40 @@
       {{> repository
```

Comment:
The partials are considered "components" because they receive attributes and can also render conditionally, when moving to the GHP repository these are written as Yew Components.

Then I think is a bit verbose to use `class="rust"` even when we already are giving `lang="Rust`!

Its much better to use the `class` `rust` inside the component if the `lang` is `Rust` instead of specifying both!

Lets give the component/partial the responsibility of rendering accordingly `lang` instead of explicitly specifying the value twice!